README.md

```diff
@@ -16,6 +16,9 @@ Note: requires MongoDB and RabbitMQ to be running to start correctly.
 
     sudo node index.js
 
+ * http://127.0.0.1:80/ui -- MonoRail Web UI
```

Comment:
Thanks @tldavies.

@WangWinson default is whatever port `on-http` uses without any configuration. In this case that is what is hard coded in `http-service.js`.

@anhou my web browser uses port 80 by default same as any other. If you try to run `on-http` by itself according to the README you run `sudo node index.js`, the reason you have to use `sudo` is because by **default** `on-http` uses 80 which requires root access to bind.

We generally run on `8080` to allow it to run alongside something that is already using 80.